### PR TITLE
Feature/92/create basket

### PR DESCRIPTION
**Pull request by GrazhdanochkaMariya**

### CODE REVIEW

File: src/apps/user/migrations/0003_alter_user_user_type.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Generated by Django 4.2.5 on 2023-09-30 20:06
```
Comment (egorvavilov) [inline src/apps/user/migrations/0003_alter_user_user_type.py]: Migration was created before your PR
Please delete it from PR

File: src/apps/basket/models.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+from django.db import models
+
+from src.apps.inventory.models import Product
+from src.apps.user.models import User
+
+
+class Basket(models.Model):
```
Comment (egorvavilov) [inline src/apps/basket/models.py]: Why do you need basket model, if we decided on pure session solution?

File: src/apps/user/admin.py [hunk 3]
```diff
@@ -4,5 +4,9 @@
 from src.apps.user.models import User, UserProfile
 
 
-admin.site.register(User, UserAdmin)
+class UserAdminExtended(UserAdmin):
```
Comment (egorvavilov) [inline src/apps/user/admin.py:7]: Please, add filter by UserType

File: src/apps/basket/forms.py [hunk 4]
```diff
@@ -0,0 +1,10 @@
+from django import forms
+
+PRODUCT_QUANTITY_CHOICES = [
```
Comment (egorvavilov) [inline src/apps/basket/forms.py:3]: Why are we explicitly set max number of items we can buy?

**Review by egorvavilov — CHANGES_REQUESTED** [review]

File: src/apps/basket/forms.py (hunk 4 above)
Comment (GrazhdanochkaMariya) [reply src/apps/basket/forms.py:3]: we can delete it, but i am not sure that this is a good choice to be able to order an endless amount of items.  Or we can leave just adding by one item

Comment (egorvavilov) [reply src/apps/basket/forms.py:3]: think of other options of how we can do that
please also make limit configurable

Comment (GrazhdanochkaMariya) [reply src/apps/basket/forms.py:3]: I see it as adding by one. On a product page you just have an option adding to the cart, for example, and in the cart you have limitation and adding just by one

Comment (egorvavilov) [reply src/apps/basket/forms.py:3]: maybe we can have positive integer field with a limit?

File: src/apps/user/forms.py [hunk 5]
```diff
@@ -0,0 +1,11 @@
+from allauth.account.forms import SignupForm
+
+from django import forms
+
+
+class UserSignupForm(SignupForm):
+    type = forms.ChoiceField(choices=[("Seller", "Seller"), ("Customer", "Customer")])
```
Comment (egorvavilov) [inline src/apps/user/forms.py]: it is not yours!!! update PR

**Review by egorvavilov — CHANGES_REQUESTED** [review]

File: src/apps/inventory/migrations/0001_initial.py [hunk 6]
```diff
@@ -43,4 +43,4 @@ class Migration(migrations.Migration):
                 ),
             ],
         ),
-    ]
+    ]
```
Comment (egorvavilov) [inline src/apps/inventory/migrations/0001_initial.py]: dont touch old migrations

**Review by egorvavilov — CHANGES_REQUESTED** [review]

**Review by egorvavilov — APPROVED** [review]

**Pull request merged**